package.json

```diff
@@ -24,6 +24,7 @@
     "clsx": "^2.1.0",
     "geist": "^1.2.2",
     "immer": "^10.1.1",
+    "install": "^0.13.0",
```

Comment:
What is install used for?

Reply:
this im honestly not sure. I think i did pnpm install to update things

Reply:
Please remove it.